src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0

```diff
@@ -0,0 +1,52 @@
+---
+subject: Java agent
+releaseDate: '2021-09-09'
+version: 7.2.0
+downloadLink: 'https://download.newrelic.com/newrelic/java-agent/newrelic-agent/7.2.0/'
+---
+
+### New features and improvements
+
+* Scala Library Instrumentation [#362](https://github.com/newrelic/newrelic-java-agent/pull/362) [#363](https://github.com/newrelic/newrelic-java-agent/pull/363)
+  * STTP versions 2 & 3  Akka-HTTP, HTTP4s and STTP core backends
+  * Cats-effect v2
+  * ZIO v1
+  * HTTP4s client & server v0.21
+  * Play 2.3-2.8
+  * Akka-HTTP v10.1 & v10.2
+  * For more information, see [Scala instrumentation](https://docs.newrelic.com/docs/agents/java-agent/frameworks/scala-installation-java/).
+
+
+* Scala API support (see PRs above)
+  * Scala APIs provided for explicit instrumentation of several of above libraries in case auto-instrumentation is not desired
+  * Cats-effect v2
+  * ZIO v1
+
+  
+* AWS v2 DynamoDB Instrumentation [#343](https://github.com/newrelic/newrelic-java-agent/pull/343)
+  * Synchronous and asynchronous AWS v2 APIs are auto-instrumented similarly to v1 APIs
+  * For more information, see [Add support for AWS SDK 2 DynamoDB sync/async clients](https://github.com/newrelic/newrelic-java-agent/issues/246)
+
+
+* GraphQL 16 Instrumentation [#396](https://github.com/newrelic/newrelic-java-agent/pull/396)
+  * Create meaningful transaction names
+  * Create meaningful spans
+  * Reporting GraphQL errors
+  * For more information, see [GraphQL for Java](https://github.com/newrelic/newrelic-java-agent/issues/356)
+
+* JFR feature causing excessive overhead when enabled [JFR #203](https://github.com/newrelic/newrelic-jfr-core/issues/203)
+  * Refactored code to use less memory.
+
+### Fixes
+The existing mongo instrumentation was partially applying when mongo reactive is being used.
```

Comment:
@twcrone, consider capitalizing  and using the entire names: "MongoDB" and "MongoDB Reactive Streams."